app/components/lunch-invites/invite/status.js

```diff
@@ -11,6 +11,7 @@ export default Ember.Component.extend({
     const id = this.get('conversation.id');
     return this.get('profiles').getMessages(id)
       .then((bool) => {
+        console.log(bool)
```

Comment:
Confused about how `getMessages(id)` returns a boolean. Looks to me like it would grab a message by it's ID.